src/firebase/api/users.js

```diff
@@ -39,6 +39,31 @@ export const signUp = async (email, password, userData) => {
   }
 };
 
+/**
```

Comment:
This looks very similar to how I implemented 'signIn' on my branch for refactoring existing APIs to use new ones. I will no longer be creating a pull request for my branch since the changes are being made here instead.